### DISCUSSION TITLE
Received tool input did not match expected schema

### DISCUSSION THREAD

**Discussion by TooMuchLag** [FlowiseAI/Flowise#2642] [Q&A]
Hey, i am creating a customer support chatbot, i am using the conversational agent with OpenAi chat model, and a pinecone vectorStore for the knowledge base, when i ask some information that's in the vector, i get the message "Received tool input did not match expected schema". Can someone help me? I've tried everything and nothing works

**Comment by marchdigital** [comment]
What LLM are you using? Try using a LLM that supports function calling (tool use) ... gpt4 (any flavour).

**Reply by danielbowne** [reply]
Know if any local models will work decent?  I been testing a few and they are just inconsistent.
